package.yaml

```diff
@@ -12,6 +12,8 @@ extra-source-files:
   - CHANGELOG.md
   - tests/golden.xml
   - tests/golden-prefixed.xml
+  - tests/golden-ghc-9.xml
+  - tests/golden-prefixed-ghc-9.xml
```

Comment:
```suggestion
  - tests/golden*.xml
```

I think this would catch all the existing and any newly-added paths, right?

Reply:
This works too. I applied it and regenerated the cabal file (the order of files changed).